test/MLOps.NET.Tests/MLOpsBuilderTests.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MLOps.NET;
+using MLOps.NET.Tests.Mocks.Storage;
+
+namespace MLOps.NET.Tests
+{
+    [TestClass]
+    public class MLOpsBuilderTests
+    {
+        [TestMethod]
+        [ExpectedException(typeof(InvalidOperationException))]
+        public void MLOpsBuilder_ThrowsIfMetaDataStoreConfiguredTwice()
+        {
+            var metaDataStore = new MockMetaDataStore();
```

Comment:
Instead of a Mock, could we use a mocking library instead, e.g. AutoFixture.AutoMoq or alike?

Reply:
Added Moq and used that